index.html

```diff
@@ -105,31 +105,162 @@ <h2>Introduction</h2>
+      </p>
+      <section>
+        <h2>Credential Header</h2>
+        <p><a data-cite="rfc7519#section-5.1">typ</a> MUST use the media type <code>vc+jwt</code>.</p>
+        <aside
+          class="example"
+          title="credential metadata (decoded JWT header)"
+        >
+          <pre data-include="./examples/vc-2.0/templates/vc+jwt/header.json" data-include-format="json"></pre>
+        </aside>
+      </section>
+      <section>
+        <h2>Credential</h2>
+        <aside
+          class="example"
+          title="credential (decoded JWT payload)"
+        >
+          <pre data-include="./examples/vc-2.0/templates/vc+jwt/claimset.json" data-include-format="json"></pre>
+        </aside>
+        <p class="note">
+          The <code>vc</code> and <code>vp</code> claims MUST NOT be present when the content type header parameter is set to <code>credential-claims-set+json</code>.
+        </p>
+      </section>
+      <section>
+        <h2>Verifiable Credential</h2>
+        <aside
+          class="example"
+          title="verifiable credential"
+        >
+          <p>An encoded verifiable credential (using external proof).</p>
+          <pre data-include="./examples/vc-2.0/vc+jwt.jose" data-include-format="text"></pre>
+        </aside>
+      </section>
+
+    </section>
+
+    <section>
+      <h2>Securing JSON-LD</h2>
+      <section>
+        <h2>With JOSE</h2>
+        <section>
+          <h2>Securing <code>application/vc+ld+json</code> with JOSE</h2>
+          <p>[[rfc7515]] MAY be used to secure this media type.</p>
+          <p><code>typ</code> MUST be <code>vc+ld+jwt</code></p>
+          <p><code>cty</code> MUST be <code>vc+ld+json</code></p>
+          <p>See <a data-cite="rfc7515#section-4.1.10">Common JOSE Header Parameters</a> for additional details.</p>
+          <section>
+            <h2>Credential Header</h2>
+            <aside
+              class="example"
+              title="verifiable credential header (decoded JWT header)"
+            >
+              <pre data-include="./examples/vc-2.0/templates/vc+ld+jwt/header.json" data-include-format="json"></pre>
+            </aside>
+          </section>
+          <section>
+            <h2>Credential</h2>
+            <aside
+              class="example"
+              title="verifiable credential (decoded JWT claimset)"
+            >
+              <pre data-include="./examples/vc-2.0/templates/vc+ld+jwt/claimset.json" data-include-format="json"></pre>
+            </aside>
+          </section>
+          <section>
+            <h2>Verifiable Credential</h2>
+            <aside
+              class="example"
+              title="verifiable credential (encoded as JWT)"
+            >
+              <pre data-include="./examples/vc-2.0/vc+ld+jwt.jose" data-include-format="json"></pre>
+            </aside>
+          </section>
+        </section>
+        <section>
+          <h2>Securing <code>application/vp+ld+json</code> with JOSE</h2>
```

Comment:
We clearly need a PR in the main spec to register this media type.

Reply:
I am happy to do one.